config/application.yml.default

```diff
@@ -120,6 +120,7 @@ idv_acuant_sdk_version_alternate: '11.8.1'
 idv_acuant_sdk_upgrade_a_b_testing_enabled: false
 idv_acuant_sdk_upgrade_a_b_testing_percent: 50
 idv_getting_started_a_b_testing: '{"welcome_default":100, "welcome_new":0, "getting_started":0}'
+idv_phone_question_a_b_testing: '{"hide_phone_question":100, "show_phone_question":0}'
```

Comment:
I am probably doing something wrong, but when I try running:

`IdentityConfig.store.idv_phone_question_a_b_testing` in the Rails console, I get `NoMethodError: undefined method 'idv_phone_question_a_b_testing'`

When I try other values from this file and from the PR linked from this one, I see a return value.

I tried `make setup && make run` on this branch first to make sure the changes were applied.

I'm probably doing something wrong, but I wanted to note it here since it was the testing plan and I'd be happy to learn how to make it work locally for me.

Reply:
It needs to be added to [the list of configuration values in the `IdentityConfig` store](https://github.com/18F/identity-idp/blob/d32eb9f286a07f931cb38f0a0ef58ca6331e87c5/lib/identity_config.rb#L86-L490) for it to be accessible in code that way.

Reply:
Thank you both, added the new variable there ([commit](https://github.com/18F/identity-idp/pull/9277/commits/dc7b35c93a5c948984e5f7d90e3e658f2e40ef40)).

Reply:
@aduth when you say that change is needed to be accessible in code that way - is there a case where you wouldn't need to add it in both places?

@charleyf Thank you for making that change! I tested it again with the changes and saw the expected value in the console.

Reply:
>@aduth when you say that change is needed to be accessible in code that way - is there a case where you wouldn't need to add it in both places?

Typically it should always be in both places. Since this ticket was already very limited in usefulness on its own, I wasn't sure just how far it was meant to be completed to be considered "functional".